sections/futures/UserInfo/OpenOrdersTable.tsx

```diff
@@ -47,36 +48,45 @@ const OpenOrdersTable: React.FC = () => {
 
 	const rowsData = useMemo(() => {
 		const ordersWithCancel = openOrders
-			.map((o) => ({
-				...o,
-				sizeTxt: formatCurrency(o.asset, o.size.abs(), {
-					currencyKey: getDisplayAsset(o.asset) ?? '',
-					minDecimals: o.size.abs().lt(0.01) ? 4 : 2,
-				}),
-				timeToExecution: countdownTimers ? countdownTimers[o.marketKey]?.timeToExecution : null,
-				timePastExecution: countdownTimers ? countdownTimers[o.marketKey]?.timePastExecution : null,
-				show: !!countdownTimers && countdownTimers[o.marketKey],
-				isStale: countdownTimers
-					? countdownTimers[o.marketKey]?.timeToExecution === 0 &&
-					  countdownTimers[o.marketKey]?.timePastExecution > 60
-					: false,
-				isExecutable: countdownTimers
-					? countdownTimers[o.marketKey]?.timeToExecution === 0 &&
-					  countdownTimers[o.marketKey]?.timePastExecution <= 60 // SET DEFAULT
-					: false,
-				totalDeposit: o.commitDeposit.add(o.keeperDeposit),
-				onCancel: () => {
-					dispatch(
-						cancelDelayedOrder({
-							marketAddress: o.marketAddress,
-							isOffchain: o.isOffchain,
-						})
-					);
-				},
-				onExecute: () => {
-					dispatch(executeDelayedOrder(o.marketAddress));
-				},
-			}))
+			.map((o) => {
+				const market = futuresMarkets.find((m) => m.marketKey === o.marketKey);
+				const order = {
+					...o,
+					sizeTxt: formatCurrency(o.asset, o.size.abs(), {
+						currencyKey: getDisplayAsset(o.asset) ?? '',
+						minDecimals: o.size.abs().lt(0.01) ? 4 : 2,
+					}),
+					timeToExecution: countdownTimers ? countdownTimers[o.marketKey]?.timeToExecution : null,
+					timePastExecution: countdownTimers
```

Comment:
There's a lot of `countdownTimers[o.marketKey]` and null checks around here making this mapping feel a bit overloaded, could do `const timer = countdownTimers?.[o.marketKey]` and have the fields accept undefined as well as null, then would be cleaner to map like:
```
timePastExecution: timer?.timePastExecution,
timeToExecution: timer?.timeToExecution,
show: !!timer
```
... etc